### PR TITLE
Caching for angular-fetched dashboard files

### PR DESCRIPTION
**Pull request by orangejenny**
per discussion last week, implementing manual caching

@dannyroberts / @calellowitz

### CODE REVIEW

File: corehq/apps/dashboard/static/dashboard/js/hq_dashboard.ng.js [hunk 1]
```diff
@@ -16,7 +16,9 @@
 
     var utils = {
         getTemplate: function (config, filename) {
-            return config.staticRoot + 'dashboard/ng_partials/' + filename;
+            // half-hearted caching: bust each week
```
Comment (benrudolph) [inline corehq/apps/dashboard/static/dashboard/js/hq_dashboard.ng.js]: haha

**Comment by benrudolph** [comment]
this does still imply that we can't get a bug fix until one week has passed. may still be worth doing the manual `v2` vs. `v3` etc. i don't have too strong of feelings though

**Comment by orangejenny** [comment]
@benrudolph Yeah, this is a cynical PR. The dashboard ng_partial files get updated so rarely, I'm betting that the next person to update them isn't going to also update this separate file, and it's better to make the update happen a few days late than not at all. If I'm wrong, the next person to update one of the ng_partial files can also switch this to a manual cache ;)

**Comment by dannyroberts** [comment]
This'll do! Agree it's a stop-gap, but one that'll get the job done for now!

**Comment by benrudolph** [comment]
:+1:

**Pull request merged**